ext/opcache/ZendAccelerator.h

```diff
@@ -123,6 +136,10 @@ typedef struct _zend_persistent_script {
 	void          *mem;                    /* shared memory area used by script structures */
 	size_t         size;                   /* size of used shared memory */
 
+	/* Some bytes must be skipped in the checksum calculation because they could've changed (legally)
+	 * since the last checksum calculation. */
+	zend_accel_skip_list_entry *mem_checksum_skip_list;
```

Comment:
I missed this in the previous review, I don't think we can even do this in a patch due to the ABI break.

Reply:
Do you mean ``zend_inheritance_cache_entry **inheritance_cache;``? We at least may try this for master. Fix for PHP-8.1 should keep binary compatibility, but I agree, it's dangerous. On the other hand targeting this PR to PHP-8.1 is also dangerous, because this is not a simple fix and it starts making changes even if ``opcache.consistency_checks == 0``

Reply:
I thought this is breaking since `sizeof(zend_persistent_script)` and `offsetof(zend_persistent_script, dynamic_members)` change. Maybe that's not the case for dynamic libraries, I don't know the details of what dynamic linking does.